### PR TITLE
Feature/countdown pane progress bar

### LINKED ISSUE #21: As a user, I should see an appropriate favicon on the tab when I view the site on desktop.

### LINKED ISSUE #22: As a user, I should see a header with the site name and my profile photo on every page.
As a user, I should see a header with the site name and my profile photo on every page.

### LINKED ISSUE #23: As a user, I can click the title of the app to be returned to my dashboard from every view.

### LINKED ISSUE #28: As a user, I should see a circular progress bar around the sundial in the countdown view.

### LINKED ISSUE #29: As a user, I should see the estimated number of days remaining until the app deploys on the countdown view.

### PR DESCRIPTION
**Pull request by AshleyOh-bit**
## Description
Added progress bar animation to Countdown view based on user's etd. Formatted etd year. Made title of app clickable to home. Wrote cypress user flow tests for countdown view. Updated favicon and added fonts.

## Concerns

-  Any bugs still present?
    There was an async issue re:setting updated etd with new year and setting progress bar percentage. Cannot recreate, may be fixed. Keep an eye out!
-  Next steps?
  center sun needs styling, as does header sun.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactor

## Checklist:
- [X] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

##Issues closed:
closes #21 
closes #22  
closes #23  
closes #28  
closes #29

### CODE REVIEW

File: src/components/App/App.js [hunk 1]
```diff
@@ -1,40 +1,82 @@
...
+  // }
+
+  // useEffect(() => {
+  //   getUser()
+  // }, [])
+
   return (
     <main>
-      <Header />
+      <Link to="/" style={{textDecoration: 'none'}}><Header /></Link>
```
Comment (tylrs) [inline src/components/App/App.js:60]: Thank you for adding this, we need to change all of the navlinks to links

File: src/components/CountdownPane/CountdownPane.js [hunk 2]
```diff
@@ -1,32 +1,84 @@
...
-      </div>
+    <section className="sundial-display">
+      <section className="release-days">
+          {error.length ? <h2>{error}</h2> : <h2> Days until release: {timeLeft.days}</h2>}
+      </section>
+      <section className="sundial">
+          <CircularProgressbar 
+            value={percentage} 
+            strokeWidth={2}
+            styles={buildStyles({strokeLinecap: 'butt',
```
Comment (tylrs) [inline src/components/CountdownPane/CountdownPane.js:56]: I am curious about the object you have in there....I assume this is part of the NPM package?

File: src/components/CountdownPane/CountdownPane.js [hunk 3]
```diff
@@ -1,32 +1,84 @@
...
+        hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+        minutes: Math.floor((difference / 1000 / 60) % 60),
+        seconds: Math.floor((difference / 1000) % 60)
+      }
+    }
+    setTimeLeft(updatedTimeLeft)
+  }
+
+  const calculateProgress = () => {
+    const total = +new Date(`${etd}`)
```
Comment (tylrs) [inline src/components/CountdownPane/CountdownPane.js:42]: Can this + operator be used in place of parseInt in other situations?

Comment (AshleyOh-bit) [reply src/components/CountdownPane/CountdownPane.js:42]: I'm not sure! It seems to be date specific.

File: src/components/CountdownPane/CountdownPane.js (hunk 2 above)
Comment (AshleyOh-bit) [reply src/components/CountdownPane/CountdownPane.js:56]: Exactly - this object is how you override the default styling for the CircleProgressBar package.

File: src/components/App/App.js [hunk 4]
```diff
@@ -1,40 +1,82 @@
...
             <NavLink className='recipient-pane' exact to='/recipients'><img src={familyTree}/></NavLink>
             <NavLink className='gallery-pane' exact to='/gallery'><img src={galleryImg}/></NavLink>
             <div className='placeholder-2'>
-                Placeholder
+              Placeholder
             </div>
           </section>
         </Route>
-        <Route exact path="/countdown" render={() => <CountdownPane />}/>
+        <Route exact path="/countdown" render={() => <CountdownPane est={user.attributes.etd} err={error}/>}/>
```
Comment (tylrs) [inline src/components/App/App.js:79]: Should the property name match the attribute? `etd={user.attributes.etd}`

File: src/components/CountdownPane/CountdownPane.js [hunk 5]
```diff
@@ -1,32 +1,89 @@
...
+        hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+        minutes: Math.floor((difference / 1000 / 60) % 60),
+        seconds: Math.floor((difference / 1000) % 60)
+      }
+    }
+    setTimeLeft(updatedTimeLeft)
+  }
+
+  const calculateProgress = () => {
+    const deathDay = +new Date(`${etd}`)
```
Comment (tylrs) [inline src/components/CountdownPane/CountdownPane.js:42]: Now i understand with this operator does!

File: src/components/CountdownPane/CountdownPane.js [hunk 6]
```diff
@@ -1,32 +1,89 @@
...
+    const birthDay = +new Date(`${dob}`)
+    const totalLife = deathDay - birthDay
+    const today =  +new Date()
+    const livedTime = today - birthDay
+    const perc = livedTime / totalLife
+
+    setPercentage((1 - perc) * 100)
```
Comment (tylrs) [inline src/components/CountdownPane/CountdownPane.js:49]: Looks like we need the progress bar to go the opposite way if its a countdown

**Review by tylrs — APPROVED** [review]

**Pull request merged**